include/odp/api/spec/timer.h

```diff
@@ -216,6 +234,20 @@ typedef struct {
 	 */
 	odp_timer_res_capability_t max_tmo;
 
+	/**
+	 * Queue types supported
+	 *
+	 * This defines the odp_queue_t type that the timer is capable of
+	 * delevering events to.
+	 * When this values is:
+	 * - `ODP_TIMER_QUEUE_TYPE_SCHED` odp_timer_alloc() can only be called
+	 *				  with sched queue.
+	 * - `ODP_TIMER_QUEUE_TYPE_PLAIN` odp_timer_alloc() can only be called
+	 *				  with plain queue.
+	 * - `ODP_TIMER_QUEUE_TYPE_ANY`   odp_timer_alloc() can be called with
+	 *				  any type of queue.
+	 */
+	odp_timer_queue_type_t queue_type;
```

Comment:
Instead of creating a new type for timer queue capability, could you use e.g. two booleans to indicate if sched/plain queues are supported with timer.

Reply:
Will remove this in the next iteration and use two booleans instead.